source/gui/addonStoreGui/controls/details.py

```diff
@@ -264,6 +264,19 @@ def _refresh(self):
 						pgettext("addonStore", "Installed version:"),
 						details._addonHandlerModel.version
 					)
+
+					self._appendDetailsLabelValue(
+						# Translators: Label for an extra detail field for the selected add-on. In the add-on store dialog.
+						pgettext("addonStore", "Minimum NVDA version:"),
+						"".join(map(str, details._addonHandlerModel.minimumNVDAVersion))

```

Comment:
There is a dedicated function to display NVDA versions. You should first add on top of your file:
`from buildVersion import formatVersionForGUI`

And then:
```suggestion
						formatVersionForGUI(*details._addonHandlerModel.minimumNVDAVersion)
```

It's better to use this function because it allows to display "2023.3" and not "2023.3.0".